### PR TITLE
Remove unused styles

### PR DESCRIPTION
**Pull request by vsn4ik**

### CODE REVIEW

**Comment by acrobat** [comment]
@vsn4ik Can you update the PR that you don't change the indentation? This will break every PR that is still open, but i'm planning to change the indents when moving to 2.0!

Just keep the removal of the unused styles here and generate the dist files and screenshots to verify that nothing breaks!

Thanks!

**Comment by vsn4ik** [comment]
PR updated, Phantomjs font looks different on Linux.

File: dist/css/bootstrap-datepicker3.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
   display: inline-block;
   border-left: 7px solid transparent;
   border-right: 7px solid transparent;
-  border-bottom: 7px solid #ccc;
+  border-bottom: 7px solid rgba(0, 0, 0, 0.15);
```
Comment (acrobat) [inline dist/css/bootstrap-datepicker3.css]: why change this to rgba?

File: dist/css/bootstrap-datepicker.standalone.css [hunk 2]
```diff
@@ -30,7 +30,7 @@
   display: inline-block;
   border-left: 7px solid transparent;
   border-right: 7px solid transparent;
-  border-bottom: 7px solid #ccc;
+  border-bottom: 7px solid #999999;
```
Comment (acrobat) [inline dist/css/bootstrap-datepicker.standalone.css]: Shouldn't this be #cccccc?

**Comment by acrobat** [comment]
Yeah I see that the font has changed, maybe you can just revert the changes on the screenshots and dist files. I will checkout this branch and try it locally (I guess my fotns are ok) and when this PR is merged I will generate the dist files and screenshots again (if needed)

**Comment by vsn4ik** [comment]
PR updated.

**Comment by acrobat** [comment]
Thanks @vsn4ik! I will try this branch later today. Then the PR should be good to merge!

**Comment by acrobat** [comment]
Tested the branch, every looks good! Thanks @vsn4ik!

**Pull request merged**